test/unit/namespaces-print-test.js

```diff
@@ -0,0 +1,14 @@
+import { printNamespaces } from "../../lib/namespaces.js";
```

Comment:
These tests sort of only test that javascript's array functionality works as it should